content/project-adopters/cloud-development-tools.md

```diff
@@ -2,4 +2,6 @@
 title: "Cloud Development Tool Project Adopters"
 date: 2019-09-10T15:50:25-04:00
 working_group: cloud-development-tools
----
\ No newline at end of file
+---
+
+{{< eclipsefdn_adopters working_group="cloud-development-tools" >}}
```

Comment:
If you are making use of this, the base page template needs to be updated to move the example code block above the section where the adopters are injected. It is most useful at the top of the page where potential adopters of this JS will see it

EDIT: Actually, reading the shortcode again, remove the block in the template that would have held the adopters as well. The shortcode injects its own container, but it isn't the first one on the page so it doesn't work correctly

Reply:
So now it becomes:

![image](https://user-images.githubusercontent.com/39588094/105246819-c6b9cf80-5b41-11eb-84a4-76cf1ede5099.png)

Reply:
Looks great!